docs/usage.md

```diff
@@ -64,6 +64,18 @@ For single-sample peaking calling without a control sample, leave the second col
 ### `--broad`
 Run MACS with the `--broad` flag. With this flag on, MACS will try to composite broad regions in BED12 ( a gene-model-like format ) by putting nearby highly enriched regions into a broad region with loose cutoff. The broad region is controlled by the default qvalue cutoff 0.1.
 
+### `--saturation`
+Running saturation analysis by sub-sampling the sequence reads of ChIP sample from 10% to 100% with 10% interval, then calling peaks with MACS.
```

Comment:
Would be nice to have a slightly longer description. So does this mean that you get ten sets of peaks for each sample? Are the intermediate subsampled files saved? Is there any plotting or analysis done on this subsampling? etc.